src/libs/actions/IOU.ts

```diff
@@ -6515,8 +6554,8 @@ function getReportFromHoldRequestsOnyxData(
         chatReport.reportID,
         chatReport.policyID ?? iouReport?.policyID ?? '',
         recipient.accountID ?? 1,
-        holdTransactions.reduce((acc, transaction) => acc + TransactionUtils.getAmount(transaction), 0),
-        getCurrency(firstHoldTransaction),
+        ((iouReport?.total ?? 0) - (iouReport?.unheldTotal ?? 0)) * (ReportUtils.isIOUReport(iouReport) ? 1 : -1),
```

Comment:
In line 6633, we pass `unheldTotal: 0,`. We also need to pass `unheldNonReimbursableTotal: 0,`

(It's related to the above point)

Reply:
Done